imports/plugins/custom/top-products/client/components/OrderList.js

```diff
@@ -58,17 +58,21 @@ class OrderList extends React.Component {
     if (Reaction.hasDashboardAccess()) {
       return (
         <div>
-          <div className="table-responsive">
+          <div className="table-responsive text-center">
             <p className="text-center">Filter your result</p>
-            <form>
+            <form className="form-inline">
```

Comment:
This doesn't seem to be related to the feature you are currently working.
Can you color more on this?

Reply:
Thanks for noting this. I had to implement the feedback received on the top products analysis feature while working on the improved media gallery feature.